src/info_module.F90

```diff
@@ -86,6 +86,21 @@ subroutine print_definitions()
       print '(g0)', 'VERBOSE is ON'
 #else
       print '(g0)', 'VERBOSE is OFF'
+#endif  
+#ifdef UNTRANSPOSE_RESTART
```

Comment:
Do you mean the VERBOSE? It is still available in Fesom.
Or the UNTRANSPOSE_RESTART? You are right, that should be removed to avoid confusion. I will do that.

Reply:
I mean `UNTRANSPOSE_RESTART` :)

Reply:
I removed it